README.md

```diff
@@ -1,93 +0,0 @@
-# tngf
-
```

Comment:
@blackcat-118 

Why remove the README file?

Reply:
All the content was generated by GitLab.